docs/source/reference/leaderboard_faq.rst

```diff
@@ -145,3 +146,26 @@ A few things you can explore to improve the speed of transcoding include:
 A few things you can explore to improve the speed of data upload/download:
 
 - Review the :doc:`bandwidth_requirements` and consider upgrading your bandwidth 
+
+How can I get historical test results for an orchestrator using the Leaderboard API ? 
+*************************************************************************************
+
+- To get aggregated stats for an orchestrator since a particular timestamp use:
```

Comment:
Describe what aggregated means?

Reply:
fixed in 74b5828d64570b09d599a456930e675c8b3a1026